src/test-run/index.js

```diff
@@ -208,6 +230,13 @@ export default class TestRun extends Session {
         if (command.type === COMMAND_TYPE.wait)
             return new Promise(resolve => setTimeout(resolve, command.timeout));
 
+        if (command.type === COMMAND_TYPE.resizeWindow ||
```

Comment:
Let's refactor it to function `isBrowserWindowCommand` and move it to commands file (like we have for `isRejectableByPageError`